xchange-bitmex/src/main/resources/bitmex.json

```diff
@@ -1,111 +1,123 @@
 {
   "currency_pairs": {
-    "BCH/BTC": {
+    "XRP/H19": {
```

Comment:
I wonder if these should be here at all. In a few months time, the H19 contracts will expire and this will be out of date.

Reply:
That's good point. I have removed everything that will expire some day.